vcap_utils/vcap_utils/backends/base_tensorrt.py

```diff
@@ -0,0 +1,243 @@
+        return final_outputs
+
+    def _prepare_post_process(self):
+        stride = 16
+        self.box_norm = 35.0
+        self.grid_h = int(self.engine_height / stride)
+        self.grid_w = int(self.engine_width / stride)
+        self.grid_size = self.grid_h * self.grid_w
+
+        self.grid_centers_h = [(i * stride + 0.5) / self.box_norm for i in range(self.grid_h)]
+        self.grid_centers_w = [(i * stride + 0.5) / self.box_norm for i in range(self.grid_w)]
+
+    def _apply_box_norm(self, o1: float, o2: float, o3: float, o4: float, x: int, y: int) -> \
+            Tuple[int, int, int, int]:
+        """
+        Applies the GridNet box normalization
+        Args:
+            o1 (float): first argument of the result
+            o2 (float): second argument of the result
+            o3 (float): third argument of the result
+            o4 (float): fourth argument of the result
+            x: row index on the grid
+            y: column index on the grid
+
+        Returns:
+            int: rescaled first argument
+            int: rescaled second argument
+            int: rescaled third argument
+            int: rescaled fourth argument
+        """
+        xmin = int((o1 - self.grid_centers_w[x]) * -self.box_norm)
+        ymin = int((o2 - self.grid_centers_h[y]) * -self.box_norm)
+        xmax = int((o3 + self.grid_centers_w[x]) * self.box_norm)
+        ymax = int((o4 + self.grid_centers_h[y]) * self.box_norm)
+        return xmin, ymin, xmax, ymax
+
+    def parse_detection_results(
+            self, results: List[List[float]],
+            resize: Resize,
+            label_map: Dict[int, str],
+            min_confidence: float = 0.0,
+    ) -> List[DetectionNode]:
+        detection_nodes: List[DetectionNode] = []
+        for class_id, class_name in label_map.items():
+            x1_idx = class_id * 4 * self.grid_size
+            y1_idx = x1_idx + self.grid_size
+            x2_idx = y1_idx + self.grid_size
+            y2_idx = x2_idx + self.grid_size
+
+            boxes = results[0]
+            for h in range(self.grid_h):
+                for w in range(self.grid_w):
+                    i = w + h * self.grid_w
+                    score = results[1][class_id * self.grid_size + i]
+                    if score >= min_confidence:
+                        o1 = boxes[x1_idx + w + h * self.grid_w]
+                        o2 = boxes[y1_idx + w + h * self.grid_w]
+                        o3 = boxes[x2_idx + w + h * self.grid_w]
+                        o4 = boxes[y2_idx + w + h * self.grid_w]
+                        xmin, ymin, xmax, ymax = self._apply_box_norm(o1, o2, o3, o4, w, h)
+                        detection_nodes.append(DetectionNode(
+                            name=class_name,
+                            coords=rect_to_coords(
+                                [xmin, ymin, xmax, ymax]
+                            ),
+                            extra_data={"detection_confidence": score},
```

Comment:
```suggestion
                            extra_data={"detection_confidence": float(score)},
```
Since BrainFrame can have issues with np.float32's, I want to make 100% sure this is a python float